lib/active_model/serializer.rb

```diff
@@ -188,7 +186,7 @@ def as_json(adapter_opts = nil)
 
     # Used by adapter as resource root.
     def json_key
-      root || _type || object.class.model_name.to_s.underscore
+      _type || object.class.model_name.to_s.underscore
```

Comment:
This is now done [here](https://github.com/rails-api/active_model_serializers/pull/1782/files#diff-6fa022966af1c59f2d2029f5cba14207R13).